doc/plugins_manual/introduction.tex

```diff
@@ -0,0 +1,46 @@
+\section{Introduction}
+
+RAVEN (Risk Analysis Virtual ENvironment) [TODO cite] is a powerful tool for risk- and uncertainty-based stochastic analysis. TODO say more.
+
+Occasionally, the applications of RAVEN become sufficiently varied and complex to warrant a special application to a particular suite of problems. These special applications may introduce new physics, new templates, new RAVEN entities, or any combination of the above. In this instance, it's worth considering whether developing a Plugin could be beneficial to you and the community.
+
+\subsection{What are Plugins?}
+
+Plugins are defined the RAVEN Software Quality Assurance documentation [TODO CITE]. They are intended to allow development that does not fit in the scope of RAVENs mission to be completed in a way that is still captured under the umbrella of RAVEN-based tools.
```

Comment:
citation

Reply:
What's the citation for our sqa material?

Reply:
I cited the user manual.